andela/ah-frontend-realers
# #166187820 User should be able to create articles

#### What does this PR do?
- create articles 

#### Description of Task to be completed?
- [x]  add create article form
- [x] add validation to forms
- [x] add tests for the component

#### How should this be manually tested?
- click [here](https://authors-frontend-staging-pr-17.herokuapp.com/create-article) to review
- first login and then click on the profile image for a dropdown menu to appear.
- select **write an article** then from there you may test our creating an article
- kindly note that you need to first login and then you should be able to create an article

**NOTE**
- after creating the article, a success message appears then after 4 seconds you are redirected to the articles page of this particular article.
- Please note that the articles page is still being worked on by @jennizalwango  so you will now see a blank page.

#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?

- Story ID - [166187820](https://www.pivotaltracker.com/story/show/166187820)
